### PR TITLE
Customer service

### PR DESCRIPTION
**Pull request by Rishabhdhiman09**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/newCustomerService.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+table, td, th {
+  border: 1px solid #E8E8E8;
+}
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/newCustomerService.css]: can you rename the file to be something like customerServiceHome.css?

File: src/main/webapp/WEB-INF/views/newCustomerService.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/newCustomerService.html]: this file could be renamed as well, I think.

File: src/main/webapp/WEB-INF/views/newCustomerService.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                        </a>
+                        <div class="dropdown-menu sm-menu">
+                            <a class="dropdown-item" href="#">Link 1</a>
+                            <a class="dropdown-item" href="#">Link 2</a>
+                            <a class="dropdown-item" href="#">Link 3</a>
+                        </div>
+                    </li>
+
+                </ul>
+                <div class="social-part">
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/newCustomerService.html]: this div can be deleted, I think. It's all hidden anyways.

File: src/main/webapp/WEB-INF/views/newCustomerService.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+            <div class="collapse navbar-collapse" id="navbarTogglerDemo03 ">
+                <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
+
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Product <span class="sr-only">(current)</span></a>
+                    </li>
+
+                    <li class="nav-item dropdown dmenu mx-2">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/newCustomerService.html]: for this dropdown's id, and the rest, why not follow something along the lines of "navbardrop1", "navbardrop2", etc?

**Review by Cactusmachete — CHANGES_REQUESTED** [review]
looks good, just a few minor changes needed

File: src/main/webapp/WEB-INF/views/customerServiceHome.css.css [hunk 5]
```diff
@@ -0,0 +1,10 @@
+table, td, th {
+  border: 1px solid #E8E8E8;
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/customerServiceHome.css.css]: why are there two css files?

File: src/main/webapp/WEB-INF/views/customerServiceHome.html [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+                        </a>
+                        <div class="dropdown-menu sm-menu">
+                            <a class="dropdown-item" href="#">Link 1</a>
+                            <a class="dropdown-item" href="#">Link 2</a>
+                            <a class="dropdown-item" href="#">Link 3</a>
+                        </div>
+                    </li>
+
+                </ul>
+                <div class="social-part">
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/customerServiceHome.html]: this div should be deleted.

File: src/main/webapp/WEB-INF/views/customerServiceHome.html [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+            <div class="collapse navbar-collapse" id="navbarTogglerDemo03 ">
+                <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
+
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Product <span class="sr-only">(current)</span></a>
+                    </li>
+
+                    <li class="nav-item dropdown dmenu mx-2">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
```
Comment (Cactusmachete) [inline src/main/webapp/WEB-INF/views/customerServiceHome.html]: navbar dropdown ids still need changing, I think

**Review by Cactusmachete — CHANGES_REQUESTED** [review]
just delete the extra file and div, and rename the navbar dowpdowns, and it should all be good

File: src/main/webapp/WEB-INF/views/customerServiceHome.css.css (hunk 5 above)
Comment (Rishabhdhiman09) [reply src/main/webapp/WEB-INF/views/customerServiceHome.css.css]: Extra file deleted

**Pull request merged**